tests/test_ijit/test_ijit_without_numba.py

```diff
@@ -19,5 +19,14 @@ def _simple_func(a, b):
         assert len(w) >= 1
         assert isinstance(_simple_func, types.FunctionType)
 
+@mock.patch.dict(sys.modules, {"numba": None})
```

Comment:
[Top-level functions should be separated by 2 blank lines](https://www.python.org/dev/peps/pep-0008/#blank-lines). Just for consistency's sake.